yetanotherco/aligned_layer
# feat(operator): merkle root check

This PR adds the merkle root check on the operator side.
Right after downloading the batch from S3, the operator will check if the merkle root of the downloaded file is the same as the expected/submitted merkle root.

To test this you can
1) run locally: anvil, aggregator, operator, batcher.
2) send proofs, normally, they should pass
3) manually go to the code and hardcode some bytecode or expected merkle root, it shoulndt pass.
